pkg/awsutils/awsutils.go

```diff
@@ -54,6 +54,11 @@ const (
 	eniClusterTagKey        = "cluster.k8s.amazonaws.com/name"
 	additionalEniTagsEnvVar = "ADDITIONAL_ENI_TAGS"
 	reservedTagKeyPrefix    = "k8s.amazonaws.com"
+	clusterNameTagKeyFormat = "kubernetes.io/cluster/%s"
```

Comment:
We already add a `cluster name` tag - https://github.com/aws/amazon-vpc-cni-k8s/blob/88c12231b93a04cbe43eef59f5adea0b3d38ade4/pkg/awsutils/awsutils.go#L837. We use this for our Cluster specific IAM policy right now. MAO will also populate this env variable by default going forward. Do we still need another tag for cluster name?